### PR TITLE
Benchmark: Only add SQLite data if the dedicated result set is incomplete

### PR DESCRIPTION
**Pull request by mweisgut**
This PR integrates the possibility to avoid loading data into SQLite if only dedicated result tables are used for the verification.

Currently, if `--verify` is used, SQLite data is loaded, even if only dedicated results are used.

### CODE REVIEW

**Comment by Bensk1** [comment]
Can we discuss that tomorrow in person?

File: scripts/setup_imdb.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def is_setup():
 
 hash_dl = hash_md5.hexdigest()
 
-if hash_dl != "79e4c71f8ec0dae17d6aa9182fdab835":
+if hash_dl != "270d3b5aa4338c7a452083cf4525c1cd":
```
Comment (Bensk1) [inline scripts/setup_imdb.py]: This change should disappear when merging the master into the feature branch.

File: src/benchmarklib/abstract_benchmark_item_runner.cpp [hunk 2]
```diff
@@ -38,6 +38,18 @@ void AbstractBenchmarkItemRunner::load_dedicated_expected_results(
   }
 }
 
+bool AbstractBenchmarkItemRunner::has_item_without_dedicated_result() {
+  if (items().size() > _dedicated_expected_results.size()) {
```
Comment (Bensk1) [inline src/benchmarklib/abstract_benchmark_item_runner.cpp]: Regarding what we discussed in person yesterday: does that mean that there is not an element in `_dedicated_expected_results` per item?

Comment (mweisgut) [reply src/benchmarklib/abstract_benchmark_item_runner.cpp]: `_dedicated_expected_results` is only a sparse vector if `load_dedicated_expected_results` was called. If it's not called, `load_dedicated_expected_results` is empty. The load-function is only called during the TPC-DS execution.

Comment (Bensk1) [reply src/benchmarklib/abstract_benchmark_item_runner.cpp]: I understand. Does that mean, we could also check for `_dedicated_expected_results.empty()`?

Comment (mweisgut) [reply src/benchmarklib/abstract_benchmark_item_runner.cpp]: The function name could be misleading then: if no items exists and thus no dedicated result, `has_item_without_dedicated_result` would return true.

Comment (Bensk1) [reply src/benchmarklib/abstract_benchmark_item_runner.cpp]: Right. We could add a comment to the current version of the method or to the header file where `_dedicated_expected_results` is declared, saying that we either have an empty or a sparse vector with the same size as `items()` to make clear that `_dedicated_expected_results.size()` is either `0` or `items().size()`.

**Pull request merged**